src/Mesh/Grids.jl

```diff
@@ -5,6 +5,7 @@ export DiscontinuousSpectralElementGrid, AbstractGrid
 export ExponentialFilter, CutoffFilter, AbstractFilter
 export dofs_per_element, arraytype, dimensionality, polynomialorder
 export referencepoints
+export grid_stretching
```

Comment:
I don't think that adding `grid_stretching` should be part of this PR since it's not used in the driver at all.

Reply:
fixed